trade-finance-manager-ui/templates/utilisation-reports/_macros/premium-payments-table.njk

```diff
@@ -38,9 +38,9 @@
         {{ tableHeader({ headerText: 'Total payments received', isNumericColumn: true, enableSorting: enablePaymentsReceivedSorting, ariaSort: 'none' }) }}
         {{ tableHeader({ headerText: 'Status', isNumericColumn: false, enableSorting: true, ariaSort: 'none' }) }}
         {% if userCanEdit %}
-          <th scope="col" class="govuk-table__header">
+          <td scope="col" class="govuk-table__header">
```

Comment:
If this is a `th` the select all checkbox aria-label gets reader out for all the items in the column, but 'Select all' describes the specific checkbox not each item in the column